apps/backend/src/queries/UserQueries.ts

```diff
@@ -157,27 +156,24 @@ export default class UserQueries {
     isValid: boolean;
     payload: AuthJwtPayload | AuthJwtApiTokenPayload | null;
   }> {
+    let payload: AuthJwtPayload | AuthJwtApiTokenPayload | null = null;
+    let isValid = false;
+
     try {
-      const payload = verifyToken<AuthJwtPayload | AuthJwtApiTokenPayload>(
-        token
-      );
+      payload = verifyToken<AuthJwtPayload | AuthJwtApiTokenPayload>(token);
+      isValid = 'user' in payload || 'accessTokenId' in payload;
 
-      if (!('user' in payload) && !('accessTokenId' in payload)) {
-        throw new GraphQLError('Unknown or malformed token');
+      if (!isValid) {
+        throw new Error('Unknown or malformed token');
```

Comment:
It depends on whether we want to have different messages and context when 'verifyToken' throws an error or the "payload" is not valid. If yes, then I change it. If not, I would use 'throw' instead of 'log' to avoid code redundancy. Let me know if you'd like different logs.

Reply:
I was thinking; having different messages with different contexts will provide more information in case the token is not valid, so I made the change. Thank you for your input! Let me know if you see something else.